test/steps/context.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package steps
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/PaesslerAG/jsonpath"
+	"github.com/cucumber/godog"
+)
+
+// registerContextSteps register all existing Context steps
+func registerContextSteps(s *godog.Suite, data *Data) {
+	s.Step(`^variable "([^"]*)" is an array of size (\d+)$`, data.variableIsArrayOfSize)
```

Comment:
can we start step with uppercase ? :)

Reply:
I see weird to read something like "And Variable ...", or "When Variable ...". For me, variable should be lowercase... What do you think?

Reply:
my point was that we have almost all our steps starting with uppercase. So I prefer consistency between them :)
@sutaakar which side are you ? :P

Reply:
Hehe yes I saw all the steps start with uppercase, but still it was weird to me to read "Then Variable". Anyway, up to you!

Reply:
or we can add an "action" verb before like `Check variable` for exmaple

Reply:
I think that we should be consistent, using `Check variable` sounds good to me

Reply:
Okis